oaknational/Oak-Web-Application
# fix: specialist overview download all button - LESQ-725

## Description

Music year: 1979

- hasDownloadableResources was hard coded to true in lesson overview query.
- Use lesson data and check copyright and existence of assets to check if download is available. constructHasDownloadableResources



## How to test

1. Go to https://deploy-preview-2342--oak-web-application.netlify.thenational.academy/teachers/specialist/programmes/communication-and-language-early-development/units/our-world-secondary-4e2e/lessons/building-anticipation-using-a-sensory-story-c8u64t#lesson-details
You should not see download all



## Checklist
